README.md

```diff
@@ -63,17 +63,17 @@ expect(increment(error)).to.deep.equal({
 Example:
 
 ```js
-const increment = createAction('INCREMENT');
+const noop = createAction('INCREMENT');
 
 // As parameter in handleAction:
-handleAction(increment, {
+handleAction(noop, {
   next(state, action) {...},
   throw(state, action) {...}
 });
 
 // As object key in handleActions:
 const reducer = handleActions({
-  [increment]: (state, action) => ({
+  [noop]: (state, action) => ({
```

Comment:
Should this not be the action creator name (i.e. 'INCREMENT' or a const) rather than the action creator function?